### PR TITLE
Update who-we-are-hiring.md

### PR DESCRIPTION
**Pull request by batemapf**
small mod to locations where 18f'rs work

### CODE REVIEW

File: pages/who-we-are-hiring.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ U.S. citizens, non-citizens who are nationals of the U.S., or people who have be
 
 ## Where are we hiring?
 
-Everywhere in the United States! If you have no experience working on remote teams or are applying for a role that requires in-person interactions with clients, we'll ask you to work out of our San Francisco or D.C. office. However, the majority of our team is distributed across the country in places like Chicago, New York, Raleigh, Tuscon, Austin, Dayton, Philadelphia, Santa Barbara, Seattle, and Portland.
+Everywhere in the United States! If you have no experience working on remote teams or are applying for a role that requires in-person interactions with clients, we'll ask you to work out of our San Francisco or D.C. office. However, the much of our team is distributed across the country in places like Chicago, New York, Raleigh, Tuscon, Austin, Dayton, Philadelphia, San Diego, Seattle, and Portland.
```
Comment (jmcarp) [inline pages/who-we-are-hiring.md]: "the much of our team" => "much of our team"

Comment (batemapf) [inline pages/who-we-are-hiring.md]: :+1:

**Comment by fureigh** [comment]
Looks good to me. @jamiealbrecht, is this list still sufficiently accurate?

**Comment by fureigh** [comment]
@jamiealbrecht says yes! Merging.

**Pull request merged**